src/index.js

```diff
@@ -87,6 +87,8 @@ async function run(request, ctx) {
     }
   }
   const html = await res.text();
+  const imgSrcPolicy = res.headers.get('x-html2md-img-src')?.split(/\s+/) || [];
+  if (imgSrcPolicy.indexOf('self') < 0) imgSrcPolicy.push('self');
```

Comment:
```suggestion
  if (imgSrcPolicy.indexOf('self') < 0) {
    imgSrcPolicy.push('self');
  }  
```

this should be detected by eslint... we don't allow single line conditions